src/Resources/contao/modules/ModuleListing.php

```diff
@@ -286,7 +292,7 @@ protected function compile()
 				'link' => $strField,
 				'href' => (ampersand($strUrl) . $strVarConnector . 'order_by=' . $arrFields[$i]) . '&amp;sort=' . $sort,
 				'title' => \StringUtil::specialchars(sprintf($GLOBALS['TL_LANG']['MSC']['list_orderBy'], $strField)),
-				'class' => $class . (($i == 0) ? ' col_first' : '') //. ((($i + 1) == count($arrFields)) ? ' col_last' : '')
+				'class' => $class . (($i == 0) ? ' col_first' : '') . ((($i + 1) == count($arrFields)) ? ' col_last' : '')
```

Comment:
I'm not sure about this change. The statement has been commented since before TYPOlight 2.6, so it doesn't seem accidental to me. @contao/developers What do you think?

Reply:
The code seems right to me. Maybe the styles for `.col_last` did cause problems?